app/controllers/profile.js

```diff
@@ -14,34 +15,176 @@ export default class ProfileController extends Controller {
   @tracked formDataKeyName = 'profile';
   @tracked showEditProfilePictureModal = false;
   @tracked title = 'Profile Details';
+  @tracked isSubmitDisabled = true;
+
+  @tracked formData = {
+    first_name: this.model.first_name,
+    last_name: this.model.last_name,
+    email: this.model.email,
+    company: this.model.company,
+    designation: this.model.designation,
+    linkedin_id: this.model.linkedin_id,
+    twitter_id: this.model.twitter_id,
+    website: this.model.website,
+  };
 
   @tracked fields = [
     {
       id: 'first_name',
-      label: 'First Name',
+      label: 'First Name*',
+      type: 'text',
+      required: true,
+      placeholder: 'First Name',
+      icon_url: 'icons/user.svg',
+      showError: false,
+      errorMessage: 'First name is required',
     },
     {
       id: 'last_name',
-      label: 'Last Name',
+      label: 'Last Name*',
+      type: 'text',
+      required: true,
+      placeholder: 'Last Name',
+      icon_url: 'icons/user.svg',
+      showError: false,
+      errorMessage: 'Last name is required',
+    },
+    {
+      id: 'email',
+      label: 'Email*',
+      type: 'email',
+      required: true,
+      placeholder: 'e.g johndoe@gmail.com',
+      icon_url: 'icons/mail.svg',
+      showError: false,
+      errorMessage: 'Valid email is required',
+      validator: this.emailValidator,
```

Comment:
Email is getting updated from here but doesn't comes with the API, therefore we can't prefill the information here.
![image](https://github.com/Real-Dev-Squad/website-my/assets/78433013/8b9fd6d1-bd59-4082-8d9a-be4c726696c9)

Can we please confirm that do backend have the email field that is getting added to DB and if it's there, can we send it in the `/users/self` API to  display it here ??

Reply:
ok, Will confirm.